src/loader.js

```diff
@@ -62,45 +112,41 @@ async function loader(content) {
   }
 
   let transformer = minimizer;
-  let parsedQuery;
-
-  if (this.resourceQuery.length > 0) {
-    parsedQuery = new URLSearchParams(this.resourceQuery);
-
-    if (parsedQuery.has("as")) {
-      if (!generator) {
-        callback(
-          new Error(
-            "Please specify the 'generator' option to use 'as' query param for generation purposes."
-          )
-        );
-
-        return;
-      }
-
-      const as = parsedQuery.get("as");
-      const presets = generator.filter((item) => item.preset === as);
+  const parsedQuery = new URLSearchParams(this.resourceQuery);
```

Comment:
Please use `this.resourceQuery.length > 0`, because `new URLSearchParams` is slow, but most of URLs don't have a query

Reply:
Done